lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -578,6 +598,7 @@ class MailboxDashBoardController extends ReloadableController {
     _connectivityStreamSubscription.cancel();
     _progressStateController.close();
     _isolateExecutor.dispose();
+    Get.delete<DownloadController>();
```

Comment:
why we need to do that? `Get` does not support us?

Reply:
This controller is not attached to the view so `Getx` will not automatically clean it.